docs/_templates/index.html

```diff
@@ -406,6 +406,12 @@ <h5 class="mb-0">
                             </div>
                             <div id="collapse5" class="collapse" role="tabpanel" aria-labelledby="faq5" data-parent="#accordion">
                                 <div class="card-body">
+                                    <p>
+                                        Another way is to install xonsh from the package manager that is supplied by the operating system. This way is <b>not</b> recommended 
+                                        because in operating systems without the <a href="https://en.wikipedia.org/wiki/Rolling_release">rolling release concept</a> 
+                                        the xonsh shell version may be very old (<a href="https://github.com/xonsh/xonsh/releases/">check latest release</a>) 
+                                        because the average release cycle for the xonsh shell is quarter.
+                                    </p>
```

Comment:
could you check the grammar of the content? especially the last statement about quarter. I am not that versed but can use ChatGPT.

Reply:
I have removed the quarter part. It is not constant and it should be called quarterly.